cmd/kubelet/app/init_windows.go

```diff
@@ -57,7 +56,7 @@ func getPriorityValue(priorityClassName string) uint32 {
 func createWindowsJobObject(pc uint32) (windows.Handle, error) {
 	job, err := windows.CreateJobObject(nil, nil)
 	if err != nil {
-		return windows.InvalidHandle, errors.Wrap(err, "windows.CreateJobObject failed")
+		return windows.InvalidHandle, fmt.Errorf("windows.CreateJobObject failed, err: %w", err)
```

Comment:
I would remove the `, err` at the end of each of these error strings.

Reply:
down